packages/remix-dev/compiler/plugins/cssFilePlugin.ts

```diff
@@ -72,9 +72,11 @@ export function cssFilePlugin(options: {
         let entry = Object.keys(outputs).find((out) => outputs[out].entryPoint);
         invariant(entry, "entry point not found");
 
-        let normalizedEntry = normalizePathSlashes(entry);
```

Comment:
If we want to guarantee that this is correct for all cases, I think its better to resolve the path relative to the project root directory specified in the Remix config. I think we can get that path by looking at `args.path` within this plugin.

Reply:
Ended up just passing through the config value so there is no guessing / accidental miss-slice of a bad path.